### PR TITLE
History accordion item validator

### PR DESCRIPTION
**Pull request by floresj**
Adds accordion validation for history subsections. Also addressed these history section related bugs:

#1902 - Where you went to school - Can't navigate back to section
#1904 - Where you went to school - Content not showing on review screed
#1887 - Where you went to school - incomplete error bug
#1895 - Where you went to school - Person who knows you details

### CODE REVIEW

File: src/components/Section/History/Education/Education.jsx [hunk 1]
```diff
@@ -49,7 +51,13 @@ export default class Education extends SubsectionElement {
                    appendLabel={i18n.t('history.education.collection.school.append')}
                    required={this.props.required}
                    scrollIntoView={this.props.scrollIntoView}>
-          <EducationItem name="Item" bind={true} required={this.props.required} scrollIntoView={this.props.scrollIntoView} />
+                   <EducationItem name="Item"
```
Comment (bmallred) [inline src/components/Section/History/Education/Education.jsx]: Missing **required** message on degree/diploma branching question

image

Comment (bmallred) [reply src/components/Section/History/Education/Education.jsx]: Also, **incomplete** error message is still showing even after all information has been supplied for education.

image

File: src/components/Section/History/Education/EducationItem.jsx [hunk 2]
```diff
@@ -111,6 +112,7 @@ export default class EducationItem extends ValidationElement {
                      onValidate={this.props.onValidate}
                      required={this.props.required}
                      scrollIntoView={this.props.scrollIntoView}
+                     onError={this.props.onError}
```
Comment (bmallred) [inline src/components/Section/History/Education/EducationItem.jsx]: **Incomplete** message is still displayed for item which has all content completed

image

Comment (bmallred) [reply src/components/Section/History/Education/EducationItem.jsx]: I'm still getting this

**Comment by bmallred** [comment]
@floresj it issue w/ the incomplete message appears to be related to the question **For this employment have any of the following happened to you in the last seven (7) years?** when answered yes and completed.

**Comment by floresj** [comment]
@bmallred I'm getting the same behavior now too. Thank ya sir!

**Pull request merged**